scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -101,7 +101,7 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
             dotnet.Restore(
                 "--verbosity", "verbose",
                 "--fallbacksource", Dirs.Corehost)
-                .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "TestAssets", "ProjectWithTests"))
+                .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "TestAssets", "ProjectsWithTests"))
```

Comment:
Just an FYI - once #2450 is merged, this won't be necessary.  Obviously you can't do anything about it now.